Description: Correct syntax error in hacking test code
Author: James Page <james.page@ubuntu.com>
Forwarded: no

diff --git a/nova/tests/unit/test_hacking.py b/nova/tests/unit/test_hacking.py
index 41cbada99f..5f35e141ca 100644
--- a/nova/tests/unit/test_hacking.py
+++ b/nova/tests/unit/test_hacking.py
@@ -517,7 +517,7 @@ class HackingTestCase(test.NoDBTestCase):
                foobar_cmd = "foobar -get -beer"
                answer = os.popen(foobar_cmd).read()
 
-               if answer == nok":
+               if answer == "ok":
                    try:
                        os.popen(os.popen('foobar -beer -please')).read()
 
